coding/traffic.hpp

```diff
@@ -9,8 +9,8 @@
 
 #include "base/checked_cast.hpp"
 
-#include "std/limits.hpp"
-#include "std/vector.hpp"
+#include <limits>
+#include <string>
```

Comment:
Just out of curiosity: why have you removed std wrappers?

Reply:
It's because we don't see where we may use the wrappers.
- for using something like STLPort? We are not planning. And it's possible to do without wrappers
- for adding something like my::make_unique for C++11? We don't need it now

BTW as far as I remember we made the decision to get rid of the wrapper when we worked together with you. 

Why do you think we need the wrappers?

Reply:
Wrappers are an easy way to port omim for any system. Good that you don't see any issues without them now.